Cabal/Distribution/Package.hs

```diff
@@ -175,7 +175,7 @@ instance NFData Module where
 -- @since 2.0
 
 data ComponentId
-    = ComponentId String
+    = ComponentId ShortText
```

Comment:
Oh yes, it goes back to the PackageName issue. Ok. ShortText is fine until we resolve that one.